src/app/modules/main/profile_section/privacy/controller.nim

```diff
@@ -67,3 +100,30 @@ proc validatePassword*(self: Controller, password: string): bool =
 method getPasswordStrengthScore*(self: Controller, password, userName: string): int = 
   return self.generalService.getPasswordStrengthScore(password, userName)
 
+proc storeToKeychain*(self: Controller, data: string) =
+  let myName = singletonInstance.userProfile.getName()
+  let value = singletonInstance.localAccountSettings.getStoreToKeychainValue()
+  if not main_constants.IS_MACOS or # Dealing with Keychain is the MacOS only feature
```

Comment:
What do you think? Won't the above-proposed idea (something like `SUPPORTS_FINGERPRINT`) help not repeating this comment over and over?